packages/dd-trace/src/appsec/index.js

```diff
@@ -12,30 +10,23 @@ const Reporter = require('./reporter')
 const web = require('../plugins/util/web')
 const { extractIp } = require('../plugins/util/ip_extractor')
 const { HTTP_CLIENT_IP } = require('../../../../ext/tags')
-const { block, loadTemplates, loadTemplatesAsync } = require('./blocking')
+const { block, setTemplates } = require('./blocking')
 
 let isEnabled = false
 let config
 
-function enable (_config) {
-  if (isEnabled) return
-
-  try {
-    loadTemplates(_config)
-    const rules = fs.readFileSync(_config.appsec.rules || path.join(__dirname, 'recommended.json'))
-    enableFromRules(_config, JSON.parse(rules))
-  } catch (err) {
-    abortEnable(err)
-  }
+function loadRecommendedJson () {
+  return require('./recommended.json')
 }
 
-async function enableAsync (_config) {
+function enable (_config) {
   if (isEnabled) return
 
   try {
-    await loadTemplatesAsync(_config)
-    const rules = await fs.promises.readFile(_config.appsec.rules || path.join(__dirname, 'recommended.json'))
-    enableFromRules(_config, JSON.parse(rules))
+    setTemplates(_config)
+
+    const rules = _config.appsec.rules || loadRecommendedJson()
```

Comment:
eeh we don't need a whole function just for that, i'd say just inline the require here no ?